### PR TITLE
Initial Lithuanian TTS

### PR DESCRIPTION
**Pull request by embar-**
Lithuanian language has several plural forms, different genders, and grammatical cases depending on context.

Implementation is based primarily on Serbian, partly on English, partly on Latvian examples, and expanded/corrected for Lithuanian needs. 

Some basic JS testing is done. More testing is welcome.

### CODE REVIEW

File: voice/lt/lt_tts.js [hunk 1]
```diff
@@ -188,9 +188,9 @@ function populateDictionary(tts) {
 	dictionary["1_hour_gen"] = tts ? "vienos valandos" : "1_hour_gen.ogg";  // [po] vienos valandos
 	dictionary["1_hour_acc"] = tts ? "vieną valandą" : "1_hour_acc.ogg";  // [per] vieną valandą
 	dictionary["hours_1_nom"] = tts ? "valanda" : "hours_1_nom.ogg";  // n-dešimt viena valanda - 21h, 31h, ... 91h
-	dictionary["hours_1_gen"] = tts ? "valandos" : "hours_1_gen.ogg";  // [po] 21-os valandos
+	dictionary["hours_1_gen"] = tts ? "valandos" : "hours_1_gen.ogg";  // [po] 21-os valandos; writting is same as "hours_2_nom", but stress differ: valandOs
```
Comment (nerijus) [inline voice/lt/lt_tts.js:191]: typo writting -> writing

**Comment by embar-** [comment]
Now I would say lt_tts is tested enough and completed. What it needed 
more to be merged?

**Comment by vshcherb** [comment]
I will merge it please test with nightly builds

**Comment by embar-** [comment]
Thanks, I downloaded from https://download.osmand.net/latest-night-build/

`OsmAnd-default.apk                                 06-Apr-2024 20:57 `
and I see lt_tts.js via archiver. 

I installed into Android, but I can not see Lithuanian (Lietuvių) option neither in navigation voice settings, neither in TTS download section (expected to see near Latvian (Latvių), Polish (Lenkų) and Norwegian (Norvegų) languages)

paveikslas
...
paveikslas

  

paveikslas
...
paveikslas

         

paveikslas

**Comment by vshcherb** [comment]
Fixed needs to be tested

**Comment by clickpik** [comment]
Google pixel 8, LT tts does not work, navigation keeps  silent. when changed to English then talks in English. osmand 4.8. on system settings tts working when pressed test phone talks in Lithuanian

**Comment by embar-** [comment]
I hope your mentioned problem  will be fixed with https://github.com/osmandapp/OsmAnd-resources/pull/1090

**Pull request merged**